spring-cloud-kubernetes-fabric8-config/src/main/java/org/springframework/cloud/kubernetes/fabric8/config/NamedSecretContextToSourceDataProvider.java

```diff
@@ -73,6 +79,12 @@ public Fabric8ContextToSourceData get() {
 				}
 				else {
 					result = dataFromSecret(secret, namespace);
+
+					if (!"".equals(source.prefix())) {
```

Comment:
actual most important change: add support for `prefix` in case of `NamedSecrets`